### PR TITLE
feat: clear selection after undo/redo

### PR DESCRIPTION
**Pull request by Innei**
#1309

### CODE REVIEW

File: packages/blocks/src/page-block/utils/bind-hotkey.ts [hunk 1]
```diff
@@ -476,6 +478,16 @@ export function bindHotkeys(
...
 }
 
+function clearSelection(page: Page) {
+  if (!page.root) return;
+  const defaultPageBlock = getDefaultPageBlock(page.root);
+
+  if ('selection' in defaultPageBlock) {
+    // this is not EdgelessPageBlockComponent
+    defaultPageBlock.selection.clear();
+    defaultPageBlock.selection.state.clear();
```
Comment (fundon) [inline packages/blocks/src/page-block/utils/bind-hotkey.ts:488]: This line should not be needed.

Comment (lawvs) [inline packages/blocks/src/page-block/utils/bind-hotkey.ts:488]:
```suggestion
```

Since `defaultPageBlock.selection.clear();` also clear state

**Review by lawvs — APPROVED** [review]
Very impressive improvement!

File: packages/blocks/src/page-block/utils/bind-hotkey.ts [hunk 2]
```diff
@@ -476,6 +478,15 @@ export function bindHotkeys(
   // Don't forget to remove hotkeys at `removeHotkeys`
 }
 
+function clearSelection(page: Page) {
+  if (!page.root) return;
+  const defaultPageBlock = getDefaultPageBlock(page.root);
+
+  if ('selection' in defaultPageBlock) {
```
Comment (himself65) [inline packages/blocks/src/page-block/utils/bind-hotkey.ts:485]: use `instanceof`

Comment (lawvs) [reply packages/blocks/src/page-block/utils/bind-hotkey.ts:485]: The `getDefaultPageBlock` has an incorrect type, I should we can fix it after

**Pull request merged**